src/showcase/index.html

```diff
@@ -48,7 +48,11 @@ <h3>{{ case.name }}</h3>
                 <h2>Made with Flutter</h2>
                 <p>
                 Flutter is used by Google and developers and organizations around the world to deliver beautiful native apps on
-                iOS and Android. Here is a small selection of apps made with Flutter.
+                iOS and Android.
+                </p>
+                <p>
+                Watch the <a href="https://youtu.be/REJDzio_h7o">Flutter App Highlight Reel</a>,
+                or see below for a small selection of apps made with Flutter.
```

Comment:
How about:

...iOS and Android. Here's a small selection of apps made with Flutter. Also see the <a href="...">Flutter app highlight reel.</a>

(That puts the link at the end, where I think it's more likely to be noticed/clicked, and also makes the wording a little less formal.)

Reply:
I like it! (I'd considered something like that, but decided against it because we write "Here ...", but then deliver the goods only after an intervening sentence. :)